spec/index.html

```diff
@@ -309,11 +393,21 @@
             <td><code>aria-activedescendant</code></td>
```

Comment:
I think the diff is a little confusing in GitHub. The table is complete, if you
expand above this it should make sense.